### PR TITLE
Set phone_number through Website SDK

### LINKED ISSUE #2599: Set phone_number through Website SDK
I checked the source code of web widget. I found there is no entry to set a contact's phone number.
image

Could you please add a phone number entry to window.$chatwood.setUser() something like following:

```
window.$chatwoot.setUser('<unique-identifier-key-of-the-user>', {
  ...
  phone_number: ''
});
```

### PR DESCRIPTION
**Pull request by aswindevps**
# Pull Request Template

## Description

Set phone_number through Website SDK

Fixes #2599

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

Test case in contact_identify_action_spec.rb

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by nithindavid — COMMENTED** [review]
FE side looks good.

File: app/actions/contact_identify_action.rb [hunk 1]
```diff
@@ -5,6 +5,7 @@ def perform
     ActiveRecord::Base.transaction do
       @contact = merge_contact(existing_identified_contact, @contact) if merge_contacts?(existing_identified_contact, @contact)
       @contact = merge_contact(existing_email_contact, @contact) if merge_contacts?(existing_email_contact, @contact)
+      @contact = merge_contact(existing_phone_number_contact, @contact) if merge_contacts?(existing_phone_number_contact, @contact)
       update_contact
     end
```
Comment (sojan-official) [inline app/actions/contact_identify_action.rb:10]: To get around with the code climate issue let's probably do something similar to this. define these private methods and move the existing logic into that

```suggestion
    ActiveRecord::Base.transaction do
      merge_if_existing_identified_contact
      merge_if_existing_email_contact
      merge_if_existing_phone_number_contact
      update_contact
    end
```

**Review by sojan-official — COMMENTED** [review]
The code looks good to me.  Let's fix the rubocop and code climate issues 

for rubocop, the following commands should be able to fix it

```
bundle exec rubocop -a
```

**Review by sojan-official — APPROVED** [review]
🎉

**Pull request merged**